bip-0360.mediawiki

```diff
@@ -0,0 +1,607 @@
+due to block space being the primary fee constraint. As a P2QRH implementation materializes, a benchmark will be added
+for performance comparison. Fortunately, SQIsign signatures are substantially faster to verify than it is to generate
+keys or to sign, which is a major consideration when a transaction need only be signed once, or a handful of times with
+PSBT, compared to being verified simultaneously on tens of thousands of nodes. Key generation may need to be cached in
+BIP-32 Hierarchical Deterministic wallets.
+
+An additional consideration is security level. Longer signature sizes provide more security. NIST has standardized five
+security levels for post-quantum cryptography. NIST security level I provides security equivalent to 128-bit keys, and
+security level V provides 256-bit security.
+
+== Test Vectors and Reference Code ==
+
+TBD
+
+== Related Work ==
+
+It is worth noting by way of comparison that
+[https://ethresear.ch/t/how-to-hard-fork-to-save-most-users-funds-in-a-quantum-emergency/18901 Vitalik Buterin's
+proposed solution] in an Ethereum quantum emergency is quite different from the approach in this BIP. His plan involves
+a hard fork of the chain, reverting all blocks after a sufficient amount of theft, and using STARKs based on BIP-32
+seeds to act as the authoritative secret when signing. These measures are deemed far too heavy-handed for Bitcoin.
+
+== References ==
+
+* [https://groups.google.com/g/bitcoindev/c/Aee8xKuIC2s/m/cu6xej1mBQAJ Mailing list discussion]
+* [https://delvingbitcoin.org/t/proposing-a-p2qrh-bip-towards-a-quantum-resistant-soft-fork/956?u=cryptoquick Delving Bitcoin discussion]
+* [https://bitcoinops.org/en/newsletters/2024/06/14/ Bitcoin OpTech newsletter]
+* [https://bitcoinops.org/en/podcast/2024/06/18/#draft-bip-for-quantum-safe-address-format Bitcoin OpTech discussion transcript]
+
+== Footnotes ==
+
+<references />
+
+== Changelog ==
+
+To help implementors understand updates to this BIP, we keep a list of substantial changes.
+
+* 2024-12-20 - Address feedback from vostrnad.
+* 2024-12-18 - Assigned BIP number.
+* 2024-12-13 - Update to use merkle tree for attestation commitment. Update LR & SR quantum attack scenarios.
+* 2024-12-06 - Update title and formatting.
+* 2024-12-03 - MediaWiki formatting fixes.
+* 2024-12-01 - Add details on attestation structure and parsing.
+* 2024-11-20 - Clarifications based on feedback from Murch. Remove some sections that are not yet ready.
+* 2024-10-21 - Replace XMSS with CRYSTALS-Dilithium due to NIST approval and size constraints.
+* 2024-09-30 - Refactor the ECC vs PoW section. Swap quitness for attestation.
+* 2024-09-29 - Update section on PoW to include partial-preimage.
+* 2024-09-28 - Add Winternitz, XMSS signatures, and security assumption types to PQC table. Omit NIST Level I table. Add spend script specification. Add revealed public key scenario table.
+* 2024-09-27 - Initial draft proposal
```

Comment:
I don’t think "Address feedback form vostrnad", "Update title and formatting", "MediaWiki formatting fixes", etc. are substantial changes that need to be tracked in the Changelog. Maybe this list could be compressed a bit.